docs/kyma/docs/017-details-local-installation.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Local installation
+type: Details
+---
+
+Simple as it may seem, the Kyma installation is a complex and dynamic process. This document extends the “031-gs-local-installation.md” guide with a detailed breakdown of the alternative installation method, i.e. the “run.sh” script.
```

Comment:
``Simple as it may seem, the Kyma installation is a complex and dynamic process. This document extends the **Local Kyma installation** guide with a detailed breakdown of the alternative installation method which is the `run.sh` script.`` 

-> why I actually made those changes: we bold doc titles + as the `kyma/docs` are visible in the UI, we rather refer to them by their titles + script formatting.

Reply:
done